README.md

```diff
@@ -20,43 +12,40 @@ The Titanium CLI requires [Node.js 0.10.x](http://nodejs.org/dist/) or newer.
 
     [sudo] npm install -g titanium
 
+After install, Titanium CLI is executable as `ti`    
```

Comment:
do we mention both `titanium` and `ti` or just `ti`?

Reply:
I don't but for some people it is easier to remember :smile: Just a general thought if we should mention it e.g. once that it is there but keep our other examples with `ti`

Reply:
I wouldn't look too much into this. I'll probably rewrite this with a proper "Usage" section.